code/modules/medical/surgery_tools.dm

```diff
@@ -1148,8 +1150,18 @@ CONTAINS:
 		src.open()
 		src.visible_message("<span class='alert'><b>[user]</b> unzips themselves from [src]!</span>")
 
-	MouseDrop(mob/user as mob)
+	MouseDrop(atom/over_object)
+		if (!over_object) return
+		if(isturf(over_object))
+			..() //Lets it do the turf-to-turf slide
+			return
+		else if (istype(over_object, /obj/screen/hud))
+			MouseDrop(usr) //Try to fold & pick up the bag instead
+			return
+		else if (!ismob(over_object))
+			return
```

Comment:
```suggestion
		else if (istype(over_object, /obj/screen/hud))
			..(usr)
		else if (over_object == usr)
			..()
		else
			return

```
Feel like emulating a new mousedrop is kind of weird, we can just make use of this one still